snap/snapcraft.yaml

```diff
@@ -0,0 +1,23 @@
+name: bigchaindb
+version: master
```

Comment:
I almost missed this comment. We have a new feature to let you use git or a script to get the version:
https://forum.snapcraft.io/t/snapcraft-version-scriptlets/106
I'm testing this new release as we speak, and I expect it to be in the archive and in the store on Monday. Later I can make a PR to see how you like the new feature, so you don't have to keep the version up to date in two places.